lib/src/services/user_service.dart

```diff
@@ -16,12 +15,36 @@ class UserService {
     return (await all()).firstWhere((user) => user.uid == uid);
   }
 
+  Future<Map<String, UserModel>> asMap() async {
```

Comment:
warum brauchst du denn diese doppelten Daten?
In jedem User steht seine uid drin